packages/core/src/sheets/worksheet.ts

```diff
@@ -86,9 +87,95 @@ export class Worksheet {
         return this._spanModel;
     }
 
+    /**
+     * Get the style of the column.
+     * @param {number} column The column index
+     * @returns {Nullable<IStyleData>|string} The style of the column
+     */
+    getColumnStyle(column: number): Nullable<IStyleData> | string {
+        return this._styles.get(this._snapshot.columnData[column]?.s);
+    }
+
+    getColumnStyleInternal(column: number): Nullable<IStyleData> {
```

Comment:
Why are there two methods here? getColumnStyleInternal and getColumnStyle seem to be essentially the same method. The `Styles.get` method also dynamically determines the type of `s`.